Automattic/wp-calypso
# Revert "Update Cookie Banner (#70601)"

This reverts commit 1b92c911fd77b40ee46aa0f95eee712a5254cf28. Related discussions in these threads:

- p1670276175778169-slack-C03N25JPCE4
- p1670277583432889-slack-C03N25JPCE4
- p1670281130099369-slack-C03N25JPCE4

### Proposed Changes

* Reverts all changes introduced in https://github.com/Automattic/wp-calypso/pull/70601

### Screenshots
#### Create Account CTA Issue

![image](https://user-images.githubusercontent.com/5414230/205762912-fdd21780-dce9-48c9-9144-bf48b65011b5.png)

#### Create Account CTA Fix

![2022-12-05_16-45](https://user-images.githubusercontent.com/5414230/205763395-ed861626-3270-43b3-aab5-0b91fbf2e6e2.png)

#### My Home Checklist Issue

<img width="1558" alt="Screen Shot 2022-12-05 at 3 35 14 PM" src="https://user-images.githubusercontent.com/5414230/205762983-5d87d060-4716-4731-bae6-b61f19733cd7.png">

#### My Home Checklist Fix

<img width="1496" alt="Screen Shot 2022-12-05 at 2 57 03 PM" src="https://user-images.githubusercontent.com/5414230/205760951-288186ff-b0df-4745-856f-4c1da790b2ef.png">

### Testing Instructions



* Check out this Calypso branch
* Spin up local dev environment
* Navigate to calypso.localhost:3000
* Verify that the my home checklist layout is correct
* Verify that the TOS Create your account CTA styling is correct

#### Pre-merge Checklist
